Code/Engine/Foundation/IO/Implementation/OpenDdlReader.cpp

```diff
@@ -265,6 +265,8 @@ void ezOpenDdlReader::ClearDataChunks()
 
 ezUInt8* ezOpenDdlReader::AllocateBytes(ezUInt32 uiNumBytes)
 {
+  uiNumBytes = ezMemoryUtils::AlignSize(uiNumBytes, static_cast<ezUInt32>(EZ_ALIGNMENT_MINIMUM));
```

Comment:
why ?

Reply:
Because otherwise the handed ou tmemory might be unaligned which will cause an bus error on android